examples/with-tailwindcss/tailwind.config.js

```diff
@@ -1,3 +0,0 @@
-module.exports = {
-	content: ['./src/**/*.{astro,html,js,jsx,svelte,ts,tsx,vue}'],
-};
```

Comment:
Does this mean we don't support a user-defined Tailwind config at the moment?

Reply:
Not in this PR, but @bholmesdev has already implemented here: #2831